CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+<!--
+ * Copyright (c) 2022,2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+-->
+
+
+# Contributing to Tractus-X
+
+Thanks for your interest in this project.
+
+## Project description
+
+The companies involved want to increase the automotive industry's
+competitiveness, improve efficiency through industry-specific cooperation and
+accelerate company processes through standardization and access to information
+and data. A special focus is also on SMEs, whose active participation is of
+central importance for the network's success. That is why Catena-X has been
+conceived from the outset as an open network with solutions ready for SMEs,
+where these companies will be able to participate quickly and with little IT
+infrastructure investment. Tractus-X is meant to be the PoC project of the
+Catena-X alliance focusing on parts traceability.
+
+* https://projects.eclipse.org/projects/automotive.tractusx
+
+## Developer resources
+
+Information regarding source code management, builds, coding standards, and
+more.
+
+* https://projects.eclipse.org/projects/automotive.tractusx/developer
+
+The project maintains the following source code repositories
+
+* https://github.com/eclipse/tractusx
+* https://github.com/catena-x
+* https://github.com/catenax-ng
+
```

Comment:
All mentioned repositories either do not exists, are not part of eclipse-tractusx or are GH orgs.

Instead you should list your own repo here (https://github.com/eclipse-tractusx/knowledge-agent).

Reply:
took the https://github.com/eclipse-tractusx/bpdm/blob/main/CONTRIBUTING.md